tool/vfx/vfxVkSection.h

```diff
@@ -363,7 +366,7 @@ class SectionPipelineOption : public Section {
   SubState &getSubStateRef() { return m_state; };
 
 private:
-  static const unsigned MemberCount = 13;
+  static const unsigned MemberCount = 15;
```

Comment:
The memberCount will increase 3 after I have add 3 more members here.
Why it's 15 and not 16? That's because I have counts the members in initialAddrTable, only 15 members are listed here. So for legacy code, even setting MemberCount = 13 doesn't have any issue, but MemberCount = 12 is more accurate here.
That's why I have set MemberCount  to 15 not 16 here.